.readthedocs.yaml

```diff
@@ -0,0 +1,32 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+    # You can also specify other tool versions:
+    # nodejs: "19"
+    # rust: "1.64"
+    # golang: "1.19"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
```

Comment:
I don't see the file docs/conf.py 

I haven't used sphinx or readthedocs before; does it happily ignore a missing conf.py file and just use defaults?  Do the defaults include building docs on the API?

Reply:
I'm actually not sure (this is my first time too). 
I think we will not immediately use sphinx at all. If I understand correctly, sphinx is used to auto-generate docs. 
Right now I'm more worried about the docs I'm writing myself. If someone wants to see the signatures of any functions they can look at the code directly (with the fairly good docstrings in place). In the future we may want to use this but right now I will just comment it out and hope that the readthedocs works without sphinx.